integration_tests/pages/barcode/reviewRecipients.ts

```diff
@@ -68,9 +75,13 @@ export default class ReviewRecipientsPage extends Page {
 
   recipientsTableBodyRows = (): PageElement => this.recipientsTable().find('.govuk-table__body tr')
 
-  addAnotherRecipientButton = (): PageElement => cy.get('[data-qa=add-another-recipient]')
+  selectAnotherRecipientYes = (): PageElement => cy.get('input[name=anotherRecipient][value=yes]')
 
-  prepareBarcodesButton = (): PageElement => cy.get('[data-qa=prepare-barcodes-button]')
+  selectAnotherRecipientNo = (): PageElement => cy.get('input[name=anotherRecipient][value=no]')
+
+  continueButton = (): PageElement => cy.get('button[data-qa="continue-button"]')
+
+  addAnotherRecipientButton = (): PageElement => cy.get('[data-qa=add-another-recipient]')
```

Comment:
Do we still have an "Add another recipient" button?

Reply:
Ah, yes, in the case where you've deleted all your previous recipients. got it 👍

Reply:
Yes if there is an empty recipients table